scanner/image/db/scripts/docker-entrypoint.sh

```diff
@@ -0,0 +1,361 @@
+#!/usr/bin/env bash
+
+### STACKROX MODIFIED - This file was copied from [the PostgreSQL Docker
```

Comment:
Is this required? Previously we were building the database from scratch, but are we doing that anymore with the new version of scanner? If not, then I think this can be changed significantly to basically be identical to the one for central-db